docs/wiki/99_Developers/Developer_Instrument_Guide.md

```diff
@@ -0,0 +1,188 @@
+# Developer's Instrument Guide
+
+> This guide reflects a typical workflow and best practices used by Loris developers at the MNI. Some specifics such as git workflow steps may not apply to all projects and implementations.
```

Comment:
Hi @haoweiqiu Could you post a screenshot of how this page (the first part) appears in Read the Docs? 
Thanks

Reply:
i.e. can you confirm this renders properly, both as a markdown and in ReadtheDocs ? thanks

Reply:
Hi @christinerogers, here's a screenshot of the first part of this page.
![image](https://user-images.githubusercontent.com/42322196/109093956-3db92980-76e7-11eb-8cde-b94edb5fbe2a.png)